### PR TITLE
deploy-to-updraft 0.0.1

### PR DESCRIPTION
**Pull request by arok**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

File: steps/deploy-to-updraft/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+title: deploy-to-updraft
+summary: |
+  Distribute your app's builds to your team and testers
+description: |
+  This step will help you to upload your app's builds to Updraft. For more information please reach [Updraft](https://getupdraft.com) site.
+website: https://github.com/appswithlove/bitrise-step-deploy-to-updraft-android
+source_code_url: https://github.com/appswithlove/bitrise-step-deploy-to-updraft-android
+support_url: https://github.com/appswithlove/bitrise-step-deploy-to-updraft-android/issues
```
Comment (tothszabi) [inline steps/deploy-to-updraft/0.0.1/step.yml:8]: These point to some private android step.

File: steps/deploy-to-updraft/0.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
+title: deploy-to-updraft
```
Comment (tothszabi) [inline steps/deploy-to-updraft/0.0.1/step.yml:1]: I would propose `Deploy to Updraft` as the title.

**Review by tothszabi — COMMENTED** [review]
I think something went wrong somewhere because in the previous PR (https://github.com/bitrise-io/bitrise-steplib/pull/4237) you mentioned that you fixed these but in this PR they appeared again.

**Comment by arok** [comment]
weird but I think I know what I did wrong (I just did a local changes in repo and than ran CLI comand to share the step, without commit and push)
Sorry about that, my bad

**Pull request closed without merging**